.editorconfig

```diff
@@ -0,0 +1,21 @@
+# http://editorconfig.org/
+
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
+
+[*.go]
+indent_style = tab
```

Comment:
With the same mindset as above, what do you think about adding `indent_size = 2`? 1/5

Reply:
How does that work with tab indents? Does that just control how tabs are displayed in the UI? I'd rather not change it if that's the case since it's more of a personal preference, and I know I like 4-wide tabs

Reply:
Oh, you'r right. Nevermind my comment.